uktrade/data-hub-frontend
# Explicitly set content type to application/json in requests

## Description of change

Fixes a bug where the activity stream was not loading.

## Test instructions

Whilst connected to a real API, go to a company's activity stream, for example, on dev, `/companies/d38a0315-db05-4c5a-ae7a-0c8763722d3e/activity` and check that the activity stream loads without any errors.

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
